README.md

```diff
@@ -23,21 +23,73 @@
+system for software projects. You use it to get a copy of the code, track
+changes to the code, and finally publish those changes for others to see and
+use. Jujutsu is unlike most other systems, because internally it abstracts
+the data model used to store your content from its user facing algorithms and
+interface. This allows it to serve as a version control interface with many
+possible physical backends, such as Git (today), while also open to other
+potential backends, like Mercurial, or even more centralized systems like
+Perforce.
+
+Today, by default, we use the Git data model to store and track content, so
+it's widely compatible with your favorite Git-based tools. All core Jujutsu
+developers use Jujutsu to develop it, right here on GitHub! But it will also
+work with your favorite Git forges, too.
+
+Jujutsu combines many distinct design features from other version control
+systems into a single user interface. Those other systems include:
+
+- **Git**: Like Git, we make an effort to [be fast][perf], from a snappy
+  UX to efficient algorithms. It also by default shares Git's data model, for
+  wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: Many Mercurial inspired features include the [revset]
+  language to select commits. There is [no explicit index][no-index] or "staging
+  area". Branches are "anonymous" like Mercurial, so you don't need to make up
+  a name for each small change. Primitives for rewriting history are powerful
+  and simple.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
+  in its model; it tracks them as if it tracked any other file. While not as
+  rigorous as patch-based systems like Pijul and Darcs, the effect is that many
+  forms of conflict resolution can be performed and propagated automatically.
+
+[perf]: https://github.com/martinvonz/jj/discussions/49
+[revset]: https://martinvonz.github.io/jj/latest/revsets/
+[no-index]: https://martinvonz.github.io/jj/latest/git-comparison/#the-index
+[conflicts]: https://martinvonz.github.io/jj/latest/conflicts/
+
+And it adds several innovative features of its own that those don't have:
+
+- **Working-copy-as-a-commit**: Changes to files are [recorded automatically][wcc]
+  as normal commits, and amended on every subsequent change. This "snapshot"
+  design simplifies the user-facing data model, simplifies internal algorithms,
+  and completely subsumes features like the Git's stash and index/staging-area.
+
+- **Operation log, undo**: Jujutsu records every operation that is performed on the
+  repository, from commits, to pulls, to pushes. This makes debugging problems like
+  "what just happened?" or "how did I end up here?" easier, *especially* when you're
+  helping your coworker answer those questions! And because everything is recorded,
+  you can undo that mistake you just made with ease. Finally: version control
+  has entered [the 1960s][undo-history]!
+
```

Comment:
nit: I don't know if the "Finally:" is meant to be read as the last item in the list of things in this paragraph or as an exclamation. If the latter, I'd probably drop the colon and maybe make it italic.

Reply:
No, it's just a cheeky way to insert some levity; it's meant to be read as "This really obvious thing now works and exists!" Not really meant as a condemnation or jab to anyone, just a little punchline :)

Reply:
Right, that's what I thought you meant. I think it might be easier to interpret it that way without the colon. The colon made me wonder if you meant it as the last item in a list "First: makes it easier to debug. Second: makes it possible to undo. Finally: VCS has entered the 1960's".

Reply:
Ah, right, we may just want to use another form of punctuation. You can't pepper in too many em dashes or semicolons. I'll touch it up.

Reply:
You can just say "Version control has finally entered the 1960s!", perhaps in a separate paragraph.